### PR TITLE
Candy Dark Color Scheme

### PR DESCRIPTION
**Pull request by rahulsharmagg**
Added Candy Dark Color Scheme

- [✔] I'm the package's author and/or maintainer.
- [✔] I have have read [the docs][1].
- [✔] I have tagged a release with a [semver][2] version number.
- [✔] My package repo has a description and a README describing what it's for and how to use it.
- [✔] My package doesn't add context menu entries. *
- [✔] My package doesn't add key bindings. **

My package is Candy-Dark Color Scheme
There are no packages like it in Package Control.
150641656-374ea307-409e-4d73-b864-52791d007765

### CODE REVIEW

**Comment by rahulsharmagg** [comment]
### Candy Dark Color Scheme
Added Vibrant Colourful Color Scheme "**Candy Dark**" **SPECIALLY** for
JavaScript + CSS + HTML &
-  Regex
- PHP 
- OTHER LANG (BASIC)

**Comment by rahulsharmagg** [comment]
**### Candy Dark Color Scheme**
Added Vibrant Colourful Color Scheme "**Candy Dark**" SPECIALLY for
JavaScript + CSS + HTML &

- > Regex
- > PHP
- > OTHER LANG (BASIC)

File: repository/c.json [hunk 1]
```diff
@@ -5021,6 +5021,16 @@
 					"tags": true
 				}
 			]
+		},
+		{
+			"name": "Candy Dark Color Scheme",
+			"details": "https://github.com/rahulsharmagg/Candy-Dark",
+			"releases": [
```
Comment (braver) [inline repository/c.json:5028]:
```suggestion
			"labels": ["color scheme"],
			"releases": [
```

**Review by braver — CHANGES_REQUESTED** [review]

**Comment by braver** [comment]
Please ensure your package is added in alphabetical order.

**Comment by rahulsharmagg** [comment]
I updated the code as needed.

**Pull request merged**